integration-tests/api/__tests__/admin/sales-channels.js

```diff
@@ -286,6 +296,19 @@ describe("sales channels", () => {
       expect(deletedSalesChannel.id).toEqual(salesChannel.id)
       expect(deletedSalesChannel.deleted_at).not.toEqual(null)
     })
+
+    it("should throw if we attempt to delete default channel", async () => {
+      const api = useApi()
+      expect.assertions(1)
+
+      try {
+        await api.delete(`/admin/sales-channels/test-channel`, adminReqConfig)
```

Comment:
suggestion: in other places we tend to use .catch instead of a block try catch. should we do the same here?

Reply:
comment: we should probably also check the response status code